app/grandchallenge/hanging_protocols/models.py

```diff
@@ -110,6 +110,10 @@ class Orientation(models.TextChoices):
                 "minimum": 0,
                 "maximum": 1,
             },
+            "slice_plane_indicator": {
+                "type": "string",
+                "enum": [port.lower() for port in ImagePort.labels],
```

Comment:
We discussed using labels in #2539, might be worth creating a separate text choice for that as the labels become part of the external API otherwise. See https://github.com/comic/grand-challenge.org/pull/2539/commits/6f58d5398676dde4bc5f9996ce6ca168c53be262 for what changed.